pimd/pim6_cmd.c

```diff
@@ -451,6 +451,34 @@ DEFPY (no_ipv6_pim_rp_prefix_list,
 	return pim_process_no_rp_plist_cmd(vty, rp_str, plist);
 }
 
+
+DEFPY (ipv6_ssmpingd,
+      ipv6_ssmpingd_cmd,
+      "ipv6 ssmpingd [X:X::X:X]$source",
+      IPV6_STR
+      "Enable ssmpingd operation\n"
```

Comment:
Please use CONF_SSMPINGD_STR instead of "Enable ssmpingd operation\n"

Reply:
Ack